server/index.js

```diff
@@ -74,11 +74,14 @@ app.post('/api/favorites/:userId/:hotelId', (req, res, next) => {
       return db.query(sql, params)
         .then(result => {
           const [fav] = result.rows;
-          res.status(201).json(fav);
+          res.json(fav);
         })
-        .catch(err => next(err));
     })
     .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'an unexpected error occurred'
+      })
```

Comment:
You don't need to do both this and `next()` if you are using error handling middleware.